app/controllers/authenticate_controller.rb

```diff
@@ -31,6 +31,16 @@ def status
     render(status_failure_response(e))
   end
 
+  def authn_jwt_status
+    Authentication::AuthnJwt::ValidateStatus.new.call(
+      authenticator_status_input: status_input("authn-jwt")
```

Comment:
why do you need to set `authn-jwt` manually? wouldn't it be the value of `params[:authenticator]`?